pandas/tests/frame/test_arithmetic.py

```diff
@@ -634,3 +634,162 @@ def test_arith_non_pandas_object(self):
         added = pd.DataFrame(df.values + val3,
                              index=df.index, columns=df.columns)
         tm.assert_frame_equal(df.add(val3), added)
+
+
+# ------------------------------------------------------------------
+# Unary
+class TestFrameUnary(object):
+
+    @pytest.mark.parametrize('typ', ['int64', 'int32', 'int16', 'int8',
```

Comment:
you can import these from pandas.conftest (we only have fixtures for the current int types, not the EA ones, though you might want to create one for combined int/EA types and use it here)

Reply:
done. Please suggest if there is better arg name.